bindings/binding_typescript_wasm/__tests__/__snapshots__/transform.js.snap

```diff
@@ -1,44 +1,44 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`transform in strip-only mode should remove declare enum 1`] = `""`;
+exports[`transform in strip-only mode should remove declare enum 1`] = `"                   "`;
```

Comment:
`declare enum` is fine and should be stripped without error.  It is a pure type-only concept.

Reply:
Yeah so the result is an empty string